src/api/Interpret.cc

```diff
@@ -57,6 +57,13 @@ bool stop;
 
 uint32_t LetFrame::id_cnt = 0;
 
+namespace {
+bool isPreLogicOption(char const * const option) {
```

Comment:
Can we move this to be part of SMTConfig?

Reply:
Changes related interpolation interface has been removed from this PR and a separate PR will be created for that.